<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Regiest</class>
 <widget class="QWidget" name="Regiest">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>466</width>
    <height>356</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="File_import_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>401</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe 宋体 Std L</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="midLineWidth">
    <number>0</number>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;基于深度学习+OpenCV的车载行人障碍物检测系统设计与实现&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>60</y>
     <width>91</width>
     <height>161</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>    账号</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>    密码</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>再次输入密码</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>   验证码</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLineEdit" name="id">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>70</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="code_display">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>190</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="pwd">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>150</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="code_input">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>200</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="regiest_check">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>270</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>注册</string>
   </property>
  </widget>
  <widget class="QPushButton" name="return_push">
   <property name="geometry">
    <rect>
     <x>254</x>
     <y>270</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QPushButton" name="reflush">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>200</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>刷新验证码</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="repwd">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>110</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="log">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>70</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>id</tabstop>
  <tabstop>repwd</tabstop>
  <tabstop>pwd</tabstop>
  <tabstop>code_input</tabstop>
  <tabstop>reflush</tabstop>
  <tabstop>regiest_check</tabstop>
  <tabstop>return_push</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
